.github/workflows/unit.yml

```diff
@@ -0,0 +1,35 @@
+on: [push, pull_request]
+
+jobs:
+  unit:
+    strategy:
+      matrix:
+        os: [ "ubuntu", "windows", "macos" ]
+        go: [ "1.14", "1.15" ]
```

Comment:
the same `.x` point as here: https://github.com/lucas-clemente/quic-go/pull/2888/files#r527797485

Reply:
I don't think that's necessary:
<img width="309" alt="image" src="https://user-images.githubusercontent.com/1478487/99823393-ea36bd00-2b86-11eb-8f3c-f2792cf8f517.png">

Reply:
Huh, that's very surprising. I'd personally still use `1.14.x` because it's very clear - Go's first 1.14 version is literally just called 1.14, not 1.14.0.